jlm/llvm/opt/alias-analyses/AliasAnalysisPrecisionEvaluator.cpp

```diff
@@ -0,0 +1,502 @@
+/*
+ * Copyright 2025 Håvard Krogstie <krogstie.havard@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include "jlm/rvsdg/Phi.hpp"
```

Comment:
<>

This is possible to configure in Clion. See here: https://youtrack.jetbrains.com/issue/CPP-41725/Option-to-disable-c-c-header-auto-import-is-missing